### PR TITLE
chore: rename organization and project

### PR DESCRIPTION
**Pull request by Pangssu**
As part of the rebranding of WEMIX Kanvas to **Kroma**, this commit includes changes to rename the name of the GitHub organization and the project to **Kroma**:

- All components in the format of kanvas-xxx are changed to kroma-xxx.
- The organization name, previously wemixkanvas, is changed to kroma-network.
- The wemixkanvas.io domain is changed to kroma.network.
- All other texts related to Kanvas are changed to Kroma.

### CODE REVIEW

File: specs/differences-from-optimism-bedrock.md [hunk 1]
```diff
@@ -25,18 +25,18 @@
...
 ### Overview
 
-Kanvas is composed of 3 nodes: `proposer`, `validator` and `vanilla`.  The following are components that
+Kroma is composed of 3 nodes: `proposer`, `validator` and `vanilla`.  The following are components that
 are used to run each node:
 
 | Node        | Components                                                                          |
 |-------------|-------------------------------------------------------------------------------------|
-| `proposer`  | `L2 EL client` + `L2 CL client` + `kanvas-batcher`                                  |
-| `validator` | `L2 EL client` + `L2 CL client` + `kanvas-validator` + (optionally `kanvas-prover`) |
```
Comment (chokobole) [inline specs/differences-from-optimism-bedrock.md:34]: Please align the table cells

**Comment by chokobole** [comment]
FYI, I can see `Optimism` keyword under `bindings/hardhat/testdata/deployments`.

**Comment by Pangssu** [comment]
> FYI, I can see `Optimism` keyword under `bindings/hardhat/testdata/deployments`.

removed legacy testdata and added sepolia and easel deployments to testdata

**Comment by seolaoh** [comment]
> npm organization `@kroma` is available, so please replace `@kroma-network` with `@kroma`.

LGTM except the name of npm org mentioned above.

**Comment by chokobole** [comment]
I added little fixes on top of your change. Please review the last change @Pangssu @seolaoh.

**Review by seolaoh — APPROVED** [review]
LGTM

**Comment by Pangssu** [comment]
Squashed

**Pull request merged**